src/components/data/MoveDialog.js

```diff
@@ -53,14 +53,13 @@ function MoveDialog(props) {
             });
             onClose();
         },
-        onError: (error) => {
-            setMoveError(error);
-        },
+        onError: setMoveError,
     });
 
     const handleMove = ({ dest }) => {
         const sources = selectedResources?.map((res) => res.path);
         if (status !== constants.LOADING && sources?.length > 0) {
+            setMoveError(null);
```

Comment:
If the user does attempt the move action again, this will clear the previous error message display while the next request is processing.